### PR TITLE
[Frontend] Create Lessons Page

### PR DESCRIPTION
**Pull request by brixtuyor**
# Related Links
- http://localhost:8000/lessons
- https://app.asana.com/0/1202403928837322/1202403928837343/f

# Description
- Lessons page markup that displays all the available lessons per category
- Progress indicator is displayed beside the category tab
- Start lesson button is disabled if previous lesson's incomplete

# Notes
- Currently working on the words learned subtask

# Updates
- Displays words learned instead of the start lesson button if the lesson is finished

# Screenshots
image
image

### CODE REVIEW

File: resources/js/pages/lessons/components/LessonCard.tsx [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+  AccordionItem,
+  AccordionPanel,
+  Box,
+  Button,
+  Heading,
+  Text,
+} from "@chakra-ui/react";
+import WordsLearned from "./WordsLearned";
+
+type LessonCardProps = {
```
Comment (roseaugusto) [inline resources/js/pages/lessons/components/LessonCard.tsx]: @brixtuyor put this in a types file

Comment (brixtuyor) [reply resources/js/pages/lessons/components/LessonCard.tsx]: Oh I forgot wait...

Comment (brixtuyor) [reply resources/js/pages/lessons/components/LessonCard.tsx]: Moved

**Review by roseaugusto — APPROVED** [review]
Approved!

**Pull request merged**